_shared/project/.gitignore

```diff
@@ -9,7 +9,8 @@ yarn-error.log
 supervisord.log
 supervisord.pid
 .DS_Store
-.devdata.env
+.devdata*
```

Comment:
Some of our projects use a `.devdata/` directory as well as or instead of a `.devdata.env` file so I've widened the ignore here to cover both cases.